### DISCUSSION TITLE
Canceling Freeplane wiki project  on the 1st of May 2022 ?

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#305] [Help] [answered]
I am considering canceling the Freeplane wiki project on the 1st of May 2022 because I have neither skills nor the capacity to run it.
If somebody wants to keep it running and at least process incoming pull requests let me know. As you can see on the example of https://github.com/freeplane/wiki/pull/8 it might require some work.

**Comment by quickfold** [comment]
I recommend that you do not cancel the project, but just automatically approve/merge any PRs that are not obvious trolls. As you say, you don't have the skills to actively manage the project by reviewing, evaluating, and managing improvements for the documentation, which is designed for new users to Freeplane rather than for complete experts. But probably anyone who is kind enough to donate their time on a serious effort to improve the docs does have those skills to some degree. Certainly doing this is better for FP's future than just cancelling it.

If you do cancel the project, then please don't ask anyone in the future to help with Freeplane documentation. This appears to be the third serious effort to document Freeplane. I don't know why the first two were abandoned, but as the creator of this last attempt, I'll just say that the reasons it were abandoned were not one-sided, but the result of miscommunication about the terms under which the collaboration would proceed. If you want to be the BDFL, and you want to have a project with documentation, but you don't want to or aren't able to effectively manage the documentation, then just give up on having documentation, because that isn't a sustainable model, as you seem to suggest with your post here.

**Comment by ofossum** [comment, marked as answer]
TL;DR
=====

* Please, no - there's so much potential and people really want it!

thanks
======

Okay, first @dpolivaev : thanks for having the courage to speak this plainly about "i don't have those capacities to run the documentation". 
Admitting something like this is not always easy.
Also, from my perspective this topic is hardly anything new and also quite painful for some people involved in this project (for example @quickfold).
Thank you as well for still being here and still pushing this issue!

I followed some of this discussion on sourceforge a bit and well… it was a long read!

the ~~need~~ potential for documentation - it's definitely there
=================================================

**I really, really, _really_ believe, Freeplane _has to have_ a documentation of some kind!**

Because: 

* it's awesome 
* the features are amazing and _a lot(!)_ 
* the capabilities beyond simple mindmapping are quite hard to understand for more casual users (i did qualitative research with it and code interviews…)
* there is a active community doing really great stuff with it (and is really, really invested!)

All in all, i think this is quite a brilliant basis for developing a good documentation. ;)

what a documentation could be
=============================

* [x] this wiki here 
* [x] some good video-tutorials elsewhere 
* [ ] screenshots
* [ ] some short infinite-cycling-gifs of functions as a first preview
* [ ] maybe also a renewed mindmap inside

practical solutions
===================

The idea of @quickfold seem to be a quite practical way in my opinion.

* have a quick look on the changes of a PR -- and if it's not crap, or trolling -- merge it
* maybe github offers different roles for a person / some persons, having the right to merge PRs only for the wiki?
* there should be a group of people willing to tackle with the documentation so that 
    * @dpolivaev should not be _the only one_ having to care about this (but also: giving up control can be -- from my own experience -- quite stressful)
* PR and contributions should be small, compact -- and not so messy like my PR 
  * but: there might be other new people which have little experience with GitHub and / or git
    + maybe a small guide or a template for people, so they can describe in a standardized way what they did in the PR -- beside git commit-messages?
      (inspiration for me was the formalization of bug-reports i noticed at [zim-wiki](https://github.com/zim-desktop-wiki/zim-desktop-wiki/issues/new/choose))

i'm capable of…
===============

I'm hardly a hardcore-user of Freeplane, even if i use it from time to time and really enjoy it -- but quite more casually than other people i've read from.
I like documentation and feel like i'm capable of writing about technical stuff in a "not-too-technical" way -- so i'd like to contribute to the wiki (obviously).

next steps
==========

* merging some small PR and see where it goes; giving some short notice if  _really_ unstructured
* consolidation of a group of people, willing to address the documentation  regularly
  * maybe: regular video-calls with topic list?
* documentation-team should have a overview over PRs and sometimes discuss bigger needed changes (re-structuring TOC for example)
* also: maybe pair-documentation using something like etherpads would work for a session and could be copy-pasted to a github-PR, if somebody has no capacity dealing with GitHub

**Reply by dpolivaev** [reply]
Awesome, I have invited you to the docu team so that you can merge your PR yourself and contribute without the need for approval.

**Reply by ofossum** [reply]
Thanks! :)

**Reply by dpolivaev** [reply]
You have not accepted the invite yet. It should have come to you per the e-mail linked with your GitHub account

**Reply by ofossum** [reply]
Now i did. ;)

**Reply by rexelbartolome** [reply]
lovely formatting 😄 but yea I agree, @quickfold also mentioned a kind of article/tutorial on how to write good documentation https://documentation.divio.com/ really informative stuff (and a relatively short read too!) I plan on contributing to the documentation soon as well so it would be great to be on the same page 😊

**Comment by macmarrum** [comment]
Dimitry wrote
> I have neither skills nor the capacity to run it.
> If somebody wants to keep it running and at least process incoming pull requests let me know.

I believe it is the right approach to free up Dimitry by having someone else taking care of the wiki project.
I agree with @quickfold that the way we can look at documentation is different from code. While incorrect (different than intended) source code has a direct and big impact on the application, incorrect/incomplete statements in documentation do little harm. Therefore docu increments can be small and the content can be changed/corrected later, if need be.

Looking at what's going on in in Freeplane's source code, I admire Dimitry's dedication and speed of implementation. I believe it would be difficult to find someone as dedicated to keep the same pace for documentation. I think we need to manage our expectations as to the wiki-project timelines.

I am glad that @ofossum joined the docu team. I'll try to devote more of my time to content revision (collaborating with @ofossum and anyone else) and to contribute more content. Recently I got involved in many initiatives and have been struggling to spend as much time as I would like to on the Freeplane wiki project. But I haven't abandoned it – I still want to complete the earlier-proposed migration from www.freeplane.org to GitHub.

**Reply by dpolivaev** [reply]
I am glad to hear it.
However, I am not going to get rid of the domain. We can mount there the GitHub wiki content instead.

**Reply by macmarrum** [reply]
Yep. I only meant the wiki content.
